node/src/main/resources/swagger-ui/openapi.yaml

```diff
@@ -2590,8 +2795,8 @@ paths:
                   format: int64
                 description: map of address <-> balance
                 example:
-                  addr1: 0
-                  addr2: 100
+                  addr1: "0"
+                  addr2: "100"
```

Comment:
There is no description of `/debug/state` response with large-significand-format